locales/de_DE.json

```diff
@@ -1303,8 +1303,8 @@
         }
       },
       "get-token": "Authentifizierung...",
-      "log-you-in": "Ich logge mich ein...",
-      "unknow-error-message": "Etwas ist schief gelaufen, bitte versuchen Sie es noch einmal."
+      "log-you-in": "Logge ein...",
```

Comment:
Bitte mit EN vergleichen. Soll `Logging in…` oder so was sein. Vllt: `Wird angemeldet …`?